sdk/typescript/test/e2e/dynamic-fields.test.ts

```diff
@@ -51,18 +53,17 @@ describe.each([{ useLocalTxnBuilder: true }])(
       expect(dynamic_fields.nextCursor).not.toEqual(null);
     });
     it('go to next cursor', async () => {
-      const dynamic_fields = await toolbox.provider.getDynamicFields(
-        parent_objectID,
-        null,
-        1
-      );
-      const dynamic_fields2 = await toolbox.provider.getDynamicFields(
-        parent_objectID,
-        dynamic_fields.nextCursor,
-        null
-      );
-      expect(dynamic_fields2.data.length).greaterThan(0);
-      expect(dynamic_fields.nextCursor).not.toEqual(null);
+      return await toolbox.provider
+        .getDynamicFields(parent_objectID, null, 1)
+        .then(async function (dynamic_fields) {
```

Comment:
```suggestion
        .then(async function (dynamicFields) {
```

The convention in TypeScript is to use camelCase for variable naming, could you fix it for all variables in this file?